core/src/main/kotlin/com/gitlab/kordlib/core/entity/User.kt

```diff
@@ -130,7 +130,7 @@ open class User(
          * Gets the avatar url in given [format] and [size], or returns null if the [format] is not supported.
          *
          * @param format    The requested image format.
-         * @param size      The requested image resolution.
+         * @param size      The requested image size.
```

Comment:
I approved this before, but don't do this: 
https://kotlinlang.org/docs/reference/coding-conventions.html#documentation-comments
> Generally, avoid using @param and @return tags. Instead, incorporate the description of parameters and return values directly into the documentation comment, and add links to parameters wherever they are mentioned. Use @param and @return only when a lengthy description is required which doesn't fit into the flow of the main text.

https://kotlinlang.org/docs/reference/coding-conventions.html#horizontal-whitespace
> As a general rule, avoid horizontal alignment of any kind. Renaming an identifier to a name with a different length should not affect the formatting of either the declaration or any of the usages.